### PR TITLE
Create pull request filter view

### PR DESCRIPTION
**Pull request by Ckitakishi**
# Description
- Create pull request filter view.
- Support dark mode.

Screen Shot 2021-11-06 at 19 14 13

### Task List
- [x] Filter view - UI <= This PR
- [x] Dark mode <= This PR
- Checkbox animation?
- Filter view - Data&Action
- Search bar

### CODE REVIEW

File: PRChecker/Shared/CommonViews/CheckBox.swift [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                Text(text)
+            }
+        }
+        .font(font)
+        .buttonStyle(PlainButtonStyle())
+    }
+
+    func toggle() -> Void {
+        isChecked.toggle()
+        // TODO: animation
```
Comment (brevansio) [inline PRChecker/Shared/CommonViews/CheckBox.swift:40]: I think it looks good,  but animation would definitely be the icing on the cake

File: PRChecker/Shared/FilterView.swift [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+//
+
+import SwiftUI
+
+struct FilterView: View {
+    var body: some View {
+        VStack {
+            Header()
+                .padding(EdgeInsets(top: 16, leading: 0, bottom: 8, trailing: 8))
+            Divider()
```
Comment (brevansio) [inline PRChecker/Shared/FilterView.swift:15]: Is this something you made or a premade class I missed? It's definitely useful!

**Review by brevansio — APPROVED** [review]
Awesome work.

File: PRChecker/Shared/FilterView.swift (hunk 2 above)
Comment (Ckitakishi) [reply PRChecker/Shared/FilterView.swift:15]: `Divider` is included in SwiftUI~!

**Pull request merged**